### DISCUSSION TITLE
How to set automatic suggestions "site wide"

### DISCUSSION THREAD

**Discussion by f3k-freek** [WeblateOrg/weblate#9858] [Q&A] [answered]
According to the documentation it's possible to set the automatic suggestions for the whole site. But I can't find how. I have found out how to set it per project.

The image in the docs is not very helpful, since it doesn't show the url nor the steps how to get to the shown webpage in the UI.

* Self hosted
* Version: 4.14

Docs: https://docs.weblate.org/en/weblate-4.14/admin/machine.html

I would specifically like to add LibreTranslate.

Am I missing something?

**Comment by nijel** [comment, marked as answer]
You can do it on /manage/machinery/. I've added mention to the docs in b92a1ff250.

**Reply by f3k-freek** [reply]
This is a lifesaver! Thank you very much!
